examples/advanced/bionemo/downstream/tap/jobs/central_tap_esm1nv/app2/custom/downstream_flip_tap.yaml

```diff
@@ -46,7 +69,7 @@ model:
       val: tap_valid
       test: tap_test
     sequence_column: "Antibody" # name of column with protein sequence in csv file
-    target_column: ["PPC"] #["3state", "resolved"] # names of label columns in csv file
+    target_column: ["SFvCSP"] #["3state", "resolved"] # names of label columns in csv file
```

Comment:
target_column seems to show twice in the new yaml file. Seems the whole `data:` section is repeated, right?

Reply:
yes I was having issues at first with the yaml and then found a solution, removed the extra data and fixed the columns so each site of the tap has their unique one